goth/runner/probe/__init__.py

```diff
@@ -225,6 +238,22 @@ async def _start_container(self) -> None:
         )
         self._logger.info("IP address: %s", self.ip_address)
```

Comment:
With the below log lines added I think we can remove this one.

Reply:
Done (318fb5fa0)